docs/Job-reference.md

```diff
@@ -4255,6 +4290,40 @@ The arguments here are in order:
 * (String) the findbugs-files to parse
 * (boolean) use the findbugs rank for the priority, default to false
 
+### Phabricator Notifier
+
+```groovy
+job {
+    publishers {
+        phabricatorNotifier {
+            commentOnSuccess(boolean comment = true)                // defaults to false
+            commentWithConsoleLinkOnFailure(boolean comment = true) // defaults to false
+            coberturaReportFile(String coverageFile)                // defaults to false
```

Comment:
this was removed in a recent refactor (it re-uses the results of the cobertura report, so it doesn't take any configuration)